Sources/Layout/UIKit/UIView+AutoLayout.swift

```diff
@@ -7,6 +7,8 @@
 //  See https://github.com/Tinder/Layout/blob/main/LICENSE for license information.
 //
 
+// swiftlint:disable file_length
+
```

Comment:
Just calling this out to ensure the line break was intentional

Reply:
Yes, intentional, thank you. It is because this a disable for the entire file, not simply the type.